study.md

```diff
@@ -13,7 +13,7 @@ material. Instead, digest what you've read and repeat it in your own voice.
 -   [RubyGems](https://en.wikipedia.org/wiki/RubyGems)
 -   [Gemfiles](http://bundler.io/gemfile.html)
 
-## What is a Gem?
+## What Is a Gem?
```

Comment:
I was following title capitalization rules for this change and not trying to place emphasis on the word, "is".